config/custom-environment-variables.json

```diff
@@ -52,6 +52,10 @@
       "termsAndConditionsEnabled": {
         "__name": "FEATURE_TERMS_AND_CONDITIONS_ENABLED",
         "__format": "json"
+      },
+      "helmetEnabled": {
+        "__name": "FEATURE_HELMET_ENABLED",
+        "__format": "json"
       }
```

Comment:
Looks fine, upto you but you can also add another "__format: 'json'" for "HELMET" too, which would allow configuring helmet via env var (optional though)